src/js/blight/BlightContainer.jsx

```diff
@@ -1,22 +1,42 @@
 import React, { Component } from "react";
+import _ from "lodash";
+
+import blightCards from "./blight-cards.json";
 
 export default class BlightContainer extends Component {
   constructor(props) {
     super(props);
     this.totalBlight = this.props.players * 2;
 
-    this.state = { numBlight: this.totalBlight };
+    this.state = { 
+      numBlight: this.totalBlight,
+      blightCard: {},
+      endGameTrigger: false
+    };
 
     this.handleAddBlight = this.handleAddBlight.bind(this);
     this.handleRemoveBlight = this.handleRemoveBlight.bind(this);
 
   }
+
   handleAddBlight(e) {
     this.setState({ numBlight: this.state.numBlight + 1 });
   }
   handleRemoveBlight(e) {
-    this.setState({ numBlight: this.state.numBlight - 1 });
+    const newBlight = this.state.numBlight - 1;
+    this.setState({ numBlight: newBlight});
+    if(newBlight <= 0 && this.state.endGameTrigger){
+      this.setState({ blightCard: { "desc": "", "name": "GAME OVER", "newblights": 0 } });
```

Comment:
Personal preference, but this is adds unnecessary complexity and coupling for what you are trying to achieve. Just have a `<div>` to display this text when the condition is true, rather than have a "fake" blight card. 
Take a look at [these couple lines](https://github.com/wabtytsai/spirit-island-helper/pull/20/files#diff-f2392796d075a75480e350ae24b84698R51)